README.md

```diff
@@ -31,16 +33,34 @@ jobs:
           image_name: ${{ env.IMAGE_NAME }}
 ```
 
-## Inputs
+
+## Properties
+
+### Inputs
 | Input | Description | Required? | Default |
 |---|---|---|---|
-| `pcc_console_url` | URL of your Prisma Cloud Compute Console | Yes |  |
+| `pcc_console_url` | URL of your Prisma Cloud Console | Yes |  |
```

Comment:
Hey @tsmithv11, I think the distinction is important in this case. Dropping Compute here would likely confuse things as we have two URLs with SaaS: one for Prisma Cloud (former RedLock) APIs and Prisma Cloud Compute (former Twistlock) APIs. "URL of your Prisma Cloud Console" tells me `app3.prismacloud.io`, not `us-west1.cloud.twistlock.com/us-3-12345`.

Reply:
That makes sense. Updated